form_designer/models.py

```diff
@@ -41,6 +41,7 @@ class FormDefinition(models.Model):
     title = models.CharField(_('title'), max_length=255, blank=True, null=True)
     body = models.TextField(_('body'), help_text=_('Form description. Display on form after title.'), blank=True, null=True)
     action = models.URLField(_('target URL'), help_text=_('If you leave this empty, the page where the form resides will be requested, and you can use the mail form and logging features. You can also send data to external sites: For instance, enter "http://www.google.ch/search" to create a search form.'), max_length=255, blank=True, null=True)
+    mail_cover_text = TemplateTextField(_('email cover text'), help_text=_('Email cover text which can be included in the default email template and in the form template.'), blank=True, null=True)
```

Comment:
This shouldn't be a `TemplateTextField`, because as far as I can see it doesn't support template syntax (it's just used as a template variable).